WooCommerce/Classes/ViewRelated/Orders/Payment Methods/Cash/CashPaymentTenderView.swift

```diff
@@ -0,0 +1,131 @@
+import SwiftUI
+import Foundation
+import WooFoundation
+
+struct CashPaymentTenderView: View {
+    @StateObject private var viewModel: CashPaymentTenderViewModel
+    @Environment(\.dismiss) var dismiss
+
+    init(viewModel: CashPaymentTenderViewModel) {
+        self._viewModel = StateObject(wrappedValue: viewModel)
+    }
+
+    var body: some View {
+        VStack {
+            ScrollView {
+                VStack(alignment: .leading, spacing: 0) {
+                    VStack(alignment: .leading, spacing: Layout.sectionVerticalSpacing) {
+                        Text(Localization.customerPaidTitle)
+                            .foregroundColor(Color(.secondaryLabel))
+                        FormattableAmountTextField(viewModel: viewModel.formattableAmountViewModel)
+                            .keyboardType(.decimalPad)
+                            .textFieldStyle(.plain)
+                            .font(.system(size: Layout.amountFontSize, weight: .bold))
+                            .simultaneousGesture(
+                                TapGesture()
+                                    .onEnded { _ in
+                                        viewModel.onCustomerPaidAmountTapped()
+                                    }
+                            )
+                    }
+                    .padding(Layout.sectionPadding)
+
+                    Divider()
+                        .padding(.leading, Layout.dividerHorizontalPadding)
+
+                    VStack(alignment: .leading, spacing: Layout.sectionVerticalSpacing) {
+                        Text(Localization.changeDueTitle)
+                            .foregroundColor(Color(.secondaryLabel))
+                        Text(viewModel.changeDue)
+                            .font(.system(size: Layout.amountFontSize, weight: .bold))
+                            .foregroundColor(Color(.label))
+                            .opacity(viewModel.hasChangeDue ? 1: 0.18)
+                    }
+                    .padding(Layout.sectionPadding)
+
+                    Toggle(Localization.addNoteToggleTitle, isOn: $viewModel.addNote)
+                        .padding(Layout.togglePadding)
+                }
+            }
+
+            VStack(spacing: 0) {
+                Divider()
+                    .frame(height: 1)
+                    .foregroundColor(Color(.separator))
+
+                Button(Localization.markOrderAsCompleteButtonTitle) {
+                    viewModel.onMarkOrderAsCompleteButtonTapped()
+                    dismiss()
+                }
+                    .buttonStyle(PrimaryButtonStyle())
+                    .disabled(!viewModel.tenderButtonIsEnabled)
+                    .padding(insets: Layout.buttonPadding)
```

Comment:
Just a cosmetic tweak, maybe this 3 lines should be indented/put inline with last }?